### PR TITLE
Document setup in readme

### PR DESCRIPTION
**Pull request by jakobhellermann**
Also moves the references to the your discord id, the slash command servers and the github app to a `constants.py` file, because they need to be changed in order to run the bot locally with different credentials.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,45 @@
...
+## Development setup
+
+1. **AWS Credentials**
+
+   Create an AWS IAM account at https://console.aws.amazon.com/ with the policy `AmazonDynamoDBFullAccess` and create an access key, then run
+   ```shell
+   aws configure
+   ```
+   with the created access key, and set your preferred default region.
+2. **Create the following dynamodb tables**
```
Comment (Kataiser) [inline README.md]: DynamoDB, not dynamodb

File: README.md [hunk 2]
```diff
@@ -4,3 +4,45 @@
...
+   | `CelesteTAS-Improvement-Tracker_contributors`  | `project_id`         | Number   |
+   | `CelesteTAS-Improvement-Tracker_sid_caches`    | `project_id`         | Number   |
+   | `CelesteTAS-Improvement-Tracker_tokens`        | `installation_owner` | String   |
+   | `CelesteTAS-Improvement-Tracker_projects`      | `project_id`         | Number   |
+   | `CelesteTAS-Improvement-Tracker_path_caches`   | `project_id`         | Number   |
+
+3. **Discord**
+
+   Create a new application at https://discord.com/developers/, go to `Bot` and click `Reset Token`. Write the token into the `bot_token` file.
+   You can invite the bot with this url: https://discord.com/api/oauth2/authorize?client_id=<your_oauth_client_id>&permissions=2147493888&scope=bot
```
Comment (Kataiser) [inline README.md]: This link like half exists, not sure if possible to fix

File: gen_token.py [hunk 3]
```diff
@@ -9,9 +9,9 @@
 
 import db
 import utils
+from constants import github_app_id
 from utils import plural
 
-
```
Comment (Kataiser) [inline gen_token.py:14]: What happened here lol

File: gen_token.py [hunk 4]
```diff
@@ -43,6 +43,10 @@ def generate_access_token(installation_owner: str, min_jwt_time: int) -> tuple:
         log.info(f"Installation ID not cached for owner \"{installation_owner}\"")
         r = requests.get('https://api.github.com/app/installations', headers=headers, timeout=30)
         utils.handle_potential_request_error(r, 200)
+
+        if r.status_code != 200:
+            raise Exception("nope")
```
Comment (Kataiser) [inline gen_token.py]: ? This is already handled tho?

**Review by Kataiser — CHANGES_REQUESTED** [review]

File: gen_token.py (hunk 4 above)
Comment (jakobhellermann) [reply gen_token.py]: oh this was just some debugging code I accidentally left in

Comment (jakobhellermann) [reply gen_token.py]: also `handle_potential_request_error` just logs a warning if the status code doesn't match but continues to execute, not sure if that's intentional

**Pull request merged**